README.md

```diff
@@ -153,23 +149,18 @@ void TokenExample()
 
     _assistant = new Assistant(_credentials);
     _assistant.VersionDate = "2018-02-16";
-    _assistant.ListWorkspaces(OnListWorkspaces, OnFail);
-}
-
-private void OnListWorkspaces(WorkspaceCollection response, Dictionary<string, object> customData)
-{
-    Log.Debug("OnListWorkspaces()", "Response: {0}", customData["json"].ToString());
+    _assistant.ListWorkspaces(OnListWorkspaces);
 }
 
-private void OnFail(RESTConnector.Error error, Dictionary<string, object> customData)
+private void OnListWorkspaces(DetailedResponse<WorkspaceCollection> response, IBMError error)
 {
-    Log.Debug("OnFail()", "Failed: {0}", error.ToString());
+    Log.Debug("OnListWorkspaces()", "Response: {0}", response.Response);
 }
 ```
 
 ### Username and password
 ```cs
-using IBM.Watson.DeveloperCloud.Services.Assistant.v1;
+using IBM.Watson.Assistant.v1;
 using IBM.Watson.DeveloperCloud.Utilities;
```

Comment:
Are utilities still in `DeveloperCloud`?

Reply:
You're right! should be `IBM.Cloud.SDK.Utilities`